### PR TITLE
apps sc: expose external redis db for harbor

### PR DESCRIPTION
**Pull request by crssnd**
**What this PR does / why we need it**: to expose external redis database for harbor

**Special notes for reviewer**:
- the migration script works with yq4 only

**Add a screenshot or an example to illustrate the proposed solution:**
redis-harbor1
redis-harbor2

**Checklist:**

- [x] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [x] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [x] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.
- Chart checklist (pick exactly one):
    - [ ] I upgraded no Chart.
    - [ ] I upgraded a Chart and determined that no migration steps are needed.
    - [x] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).

### CODE REVIEW

**Review by robinAwallace — APPROVED** [review]
LGTM :+1: We should wait until yq merge task is complete.

**Review by Xartos — APPROVED** [review]

**Review by raviranjanelastisys — APPROVED** [review]

**Pull request merged**
